flow-report/src/common.tsx

```diff
@@ -99,7 +98,7 @@ const FlowStepThumbnail: FunctionComponent<{
       return <FlowStepAnimatedThumbnail frames={frames} width={width} height={height} />;
     }
   } else {
-    thumbnail = fullPageScreenshot?.screenshot.data;
+    thumbnail = lhr.fullPageScreenshot?.screenshot.data;
```

Comment:
This is an `lhr` not a report result so it won't be backwards compatible

Reply:
Good catch